### PR TITLE
Remove the PushManager.dispatchForChid method

### LINKED ISSUE #877: Push Component should return None instead of blank strings
for `components/push/ffi/src/lib.rs`

`push_verify_connection` and `push_dispatch_for_chid` should return None values.
This may require changing to `Result<Option<String>>` as well as tweaking the `rustCall`  functions in `PushManager.kt`.

Need to ping @thomcc for help on this one.

┆Issue is synchronized with this [Jira Story](https://jira.allizom.org/browse/SYNC-144)

### PR DESCRIPTION
**Pull request by eoger**
Fixes #877.
Diff for `PushManager.kt` looks weird because I fixed the indentation of a nearby method, please review with `?w=1` appended to the URL.

### CODE REVIEW

File: components/push/ffi/src/lib.rs [hunk 1]
```diff
@@ -163,28 +163,6 @@ pub extern "C" fn push_decrypt(
 }
 // TODO: modify these to be relevant.
```
Comment (jrconlin) [inline components/push/ffi/src/lib.rs]: Can probably drop this TODO as well.

**Comment by eoger** [comment]
From @jonalmeida :
> It looks like we need to keep it according to the API docs: https://bugzilla.mozilla.org/show_bug.cgi?id=1343678#c7

**Comment by jonalmeida** [comment]
To clarify for visibility, I did some investigation and we don't need this today, but it will help us for retrieving the scope when we support WebPush.

**Comment by eoger** [comment]
Thanks! Will re-open another PR to fix the method.

**Pull request closed without merging**